journal-writings/modernized/04-chapter-2.adoc

```diff
@@ -47,9 +47,9 @@ in a way which became my youth and his years: and though it was a hard task to m
 it was well taken, and, I believe, was useful to us both.
 
 Having now been several years with my employer,
-and he doing less at merchandize than heretofore,
+and he doing less at merchandise than heretofore,
 I was thoughtful of some other way of business;
-perceiving merchandize to be attended with much cumber,
+perceiving merchandise to be attended with much cumber,
```

Comment:
Yeah, this is tough. But I sort of lean towards carrying the change through here as well. I would propose:

```ASCIIDOC
Having now been several years with my employer,
and he doing less business as a merchant than heretofore,
I was thoughtful of some other way of business;
perceiving the occupation of a merchant to be attended with much cumber,
in the way of trading in these parts.
```

Does that seem too strong of a change? What do you think Julie? @Henderjay?

Reply:
Here's what I think.... I think BOTH of you guys are much, much smarter than I am ... and I am not attached to tenses, variants, certain spellings, grammar-ish things AT ALL! :)  [Except maybe commas... three dots, and exclamation points! I love all three of those! I know I usually have an opinion on almost everything (!) but I am pretty sure I feel extremely neutral about these grammatical sorts of things and wholly trust in your opinions and suggestions in this editing stuff!! Please feel free to just tell  me what to change and I'll change it.... and unless you give your reasons for each change to everyone because that is just what you like/prefer to do, you really really don't have to tell me your rationale at all! I trust you both implicitly! :) I also think that I kinda wish I didn't have anything else to do (as in cook, clean, work, etc) because I like working on this stuff! It's a bit addicting, which is probably a problem! :)